src/helper.py

```diff
@@ -5,6 +5,10 @@
 import botocore
 import pypdf
 import re
+import json
+import csv
+import openai
```

Comment:
please add openai package under requirements.txt so that dev container can install it for everyone whos using it